CONTRIBUTORS.txt

```diff
@@ -21,10 +22,16 @@ needs to be listed here.
 - Andrew Naylor <andrew_naylor@apple.com>
 - Andrius Aucinas <aaucinas@apple.com>
 - Ankit Aggarwal <ankit_aggarwal@apple.com>
+- Antoine Cœur <coeur@gmx.fr>
+- Antwan van Houdt <jabwd@exurion.com>
 - Arjun Nayini <aanayini@gmail.com>
 - Artem Redkin <aredkin@apple.com>
+- Austin <Austinpayne@users.noreply.github.com>
 - Bas Broek <bas@basbroek.nl>
+- Bastian Inuk Christensen <bastian@inuk.blog>
 - Ben Cohen <ben_cohen@apple.com>
+- BenedictSt <52418145+BenedictSt@users.noreply.github.com>
+- Boby <73424060+bobychaudhary@users.noreply.github.com>
```

Comment:
@bobychaudhary  if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
(also if you would like, we can add a real name here if you let me know what you'd like it to read)